app/web/features/search/SearchResultsList.test.tsx

```diff
@@ -80,12 +88,20 @@ describe("SearchResultsList", () => {
   it("Is blank with no selection or query", () => {
     render(
       <SearchResultsList
-        updateMapBoundingBox={() => jest.fn()}
+        isLoading={false}
+        results={undefined}
+        error={undefined}
+        hasNext={undefined}
+        fetchNextPage={() => {}}
```

Comment:
You don't need to pass props with `undefined` here, you can just remove the props entirely if you don't need them for this component. It will get an undefined value be default then.